src/data/set/finite.lean

```diff
@@ -119,51 +119,93 @@ protected lemma finite_or_infinite (s : set α) : s.finite ∨ s.infinite := em
 
 /-! ### Basic properties of `set.finite.to_finset` -/
 
-section finite_to_finset
-variables {s t : set α}
+namespace finite
+variables {s t : set α} {a : α}
 
-@[simp] lemma finite.coe_to_finset {s : set α} (h : s.finite) : (h.to_finset : set α) = s :=
-@set.coe_to_finset _ s h.fintype
-
-@[simp] theorem finite.mem_to_finset {s : set α} (h : s.finite) {a : α} : a ∈ h.to_finset ↔ a ∈ s :=
+@[simp] protected lemma mem_to_finset (h : s.finite) : a ∈ h.to_finset ↔ a ∈ s :=
 @mem_to_finset _ _ h.fintype _
 
-@[simp] theorem finite.nonempty_to_finset {s : set α} (h : s.finite) :
-  h.to_finset.nonempty ↔ s.nonempty :=
+@[simp] protected lemma coe_to_finset (h : s.finite) : (h.to_finset : set α) = s :=
+@coe_to_finset _ _ h.fintype
+
+@[simp] protected lemma to_finset_nonempty (h : s.finite) : h.to_finset.nonempty ↔ s.nonempty :=
```

Comment:
Are you sure this shouldn't be `nonempty_to_finset`?

Reply:
It was already `set.to_finset_nonempty` and `nonempty` is mostly put as a suffix. See `set.mul_nonempty`, `set.image2_nonempty_iff`. If you want it the other way around, I'd rather make a big PR that fixes everything at once rather than fiddle locally.